theurgy.lic

```diff
@@ -744,11 +744,24 @@ class TheurgyActions
 
   def wine
     walk_to_altar_or_prayer_mat
-    DRCI.get_item_safe('wine', @theurgy_supply_container)
-    if @prayer_mat
-      fix_standing
-      bput("kneel #{@prayer_mat}", 'You humbly kneel')
+    return unless DRCI.get_item?("wine", @theurgy_supply_container)
+      if @prayer_mat
+        fix_standing
+        bput("kneel #{@prayer_mat}", 'You humbly kneel')
+      else
+        return
+      end
+    else
```

Comment:
I think this `else` should have been removed from the `return unless` was refactored